src/constants/CommentTypes.js

```diff
@@ -0,0 +1,7 @@
+// COMMENTS ENTITY ACTIONS
```

Comment:
I'm thinking how we can better approach this as the `ActionTypes` file has started growing really fast. Maybe `ActionTypes.js` should import from other files and then export everything ?

Reply:
There should be an index.js in the root that exports each individual action types(CommentTypes, FIlterTypes aso). This way you have the option to import in other files either all actions types or the ones specific for the desired goal. We should not aggregate all in ActionTypes. Pls consider this in the refactoring. Actually I am going to put this comment in there such that we won't forget. Thanks